sui/deploy-upgradable.js

```diff
@@ -0,0 +1,175 @@
+const { Command, Option } = require('commander');
+const { TxBuilder, updateMoveToml } = require('@axelar-network/axelar-cgp-sui');
+const { bcs } = require('@mysten/sui.js/bcs');
+const { fromB64, toB64 } = require('@mysten/bcs');
+const { saveConfig, printInfo, validateParameters, prompt, writeJSON } = require('../evm/utils');
+const { addBaseOptions } = require('./cli-utils');
+const { getWallet } = require('./sign-utils');
+const { loadSuiConfig } = require('./utils');
+
+async function upgradePackage(client, keypair, packageName, packageConfig, builder, options) {
+    const { modules, dependencies, digest } = await builder.getContractBuild(packageName);
+    const { policy, offline, sender } = options;
+
+    const upgradeCap = packageConfig.objects?.UpgradeCap;
+    const digestHash = options.digest ? fromB64(options.digest) : digest;
+
+    validateParameters({ isNonEmptyString: { upgradeCap, policy }, isNonEmptyStringArray: { modules, dependencies } });
+
+    const tx = builder.tx;
+    const cap = tx.object(upgradeCap);
+
+    const ticket = tx.moveCall({
+        target: `0x2::package::authorize_upgrade`,
```

Comment:
avoid magic constants
```suggestion
        target: `${suiPackage}::package::authorize_upgrade`,
```

Reply:
done